nemo_skills/pipeline/generate.py

```diff
@@ -113,6 +113,36 @@ class GenerationType(str, Enum):
     GenerationType.math_judge: get_math_judge_cmd,
 }
 
+def configure_client(generation_type, server_gpus, server_type, server_address, server_port, server_nodes, model, server_args, extra_arguments):
+    if server_address is None:  # we need to host the model
+        server_port = get_free_port()
+        assert server_gpus is not None, "Need to specify server_gpus if hosting the model"
+        # Note: for nemo reward models, the port is hard-coded to 5000 in the
+        # get_reward_server_command function. Since RM has a proxy server on 5000
+        # and an actual GPU is being loaded on port 5001, we need to wait for 5001
+        # to come online before sending requests
+
+        # if generation_type == GenerationType.reward and server_type == SupportedServers.nemo:
```

Comment:
should we remove this commented block?

Reply:
Removed in [f157a1f](https://github.com/NVIDIA/NeMo-Skills/pull/298/commits/f157a1f5a59b8625dd5a55704026d85b91f58640)